src/frontend/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "record-expunge-pdx",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:5000",
```

Comment:
I have question. Having separate frontend and backend frameworks is new to me, and so connecting the two is completely new to me. Hypothetically if this was being deployed we would also need to set a production proxy too, I assume? if so where would we set this?

Reply:
Here's how this typically works in prod:

When deploying the frontend code, it gets transpiled, minified, etc. (in our case, by webpack) to a set of static assets, at minimum, just an empty HTML page with a script tag, where the script tag links to a big JS file that's our React app.

When someone visits the home page in prod, the "home" route in Flask serves that HTML page, the client downloads the JS file, and then the React app boots, and most of the remaining requests to the server are generated by the React app.

The simplest way to do this is: when deploying the backend, have a task that runs on the filesystem to build the frontend code. If the built frontend assets are in the filesystem that gets deployed, then the "home" route in Flask would look like:

    with open('home.html', 'r') as home_file:
      html_text = home_file.read()
      return html_text

There is no proxy in production. We have a proxy in dev so that we can run an additional server for the frontend code, which massively speeds up the dev experience by incorporating new changes to the React code when you edit a file (without recompiling the whole JS asset) and autoreloading the page so you don't have to manually refresh to see the changes.

Reply:
I see. Thanks for the detailed explanation!